### PR TITLE
[Release 1.6.0] Updating DNS powershell to new SDK

### PR DESCRIPTION
**Pull request by vladrai**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@vladrai__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by hovsepm** [comment]
@azuresdkci add to whitelist

**Comment by hovsepm** [comment]
>     [Cmdlet(VerbsCommon.Remove, "AzureRmDnsZone", SupportsShouldProcess = true),

please implement ShouldProcess correctly. 

---
Refers to: src/ResourceManager/Dns/Commands.Dns/Zones/RemoveAzureDnsZone.cs:27 in 3d49692. [](commit_id = 3d49692825b20f0c4635043821d4dbcab8c87871, deletion_comment = False)

File: src/ResourceManager/Dns/Commands.Dns.Test/Commands.Dns.Test.csproj [hunk 1]
```diff
@@ -191,6 +215,7 @@
     <None Include="..\..\Common\Commands.ScenarioTests.ResourceManager.Common\AzureRM.Resources.ps1">
       <Link>ScenarioTests\AzureRM.Resources.ps1</Link>
     </None>
+    <None Include="app.config" />
     <None Include="MSSharedLibKey.snk" />
```
Comment (hovsepm) [inline src/ResourceManager/Dns/Commands.Dns.Test/Commands.Dns.Test.csproj]: please remove app.config from csproj.

**Comment by hovsepm** [comment]
on-demand: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/1003/

**Comment by hovsepm** [comment]
@vladrai on-demand job failed. you need to set location of the session records relative to your Assembly location. Take a look here - https://github.com/Azure/azure-powershell/blob/83203f0c2ab4988661f570991b4b26a6c8ee8338/src/ResourceManager/RedisCache/Commands.RedisCache.Test/ScenarioTests/RedisCacheController.cs#L71

**Comment by vladrai** [comment]
Updated PR to have script directory setup

**Comment by hovsepm** [comment]
new on-demand: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/1004/

**Pull request merged**